compiler/apps/playground/__tests__/e2e/__snapshots__/page.spec.ts/01-user-output.txt

```diff
@@ -1,4 +1,5 @@
-function TestComponent(t0) {
+import { c as _c } from "react/compiler-runtime";
```

Comment:
Note that we now print inserted imports. Other program level changes (e.g. `Options.gating`, outlining) should work as expected without hardcoding